src/app/util/BUILD.gn

```diff
@@ -15,22 +15,35 @@
 import("//build_overrides/chip.gni")
 import("${chip_root}/src/app/common_flags.gni")
 
+source_set("null-utils") {
```

Comment:
Trying to avoid filler words of utils/helpers/core/support as they are vague.

When I look at `null` only, it is not obvious why "odd sized integers" is related to null. Can we rename?

I propose one of `primitives` or `basic-types`